js/ui/html_editor/utils/image_uploader_helper.js

```diff
@@ -477,7 +472,7 @@ class FileStrategy extends BaseStrategy {
                 this.data = data;
                 this.onFileSelected();
             },
-            onUploaded: e => this.serverUpload(e)
+            onUploaded: e => this.onUploaded(e)
```

Comment:
We don't call `this.closeDialogPopup(data);` anymore. Is it ok?

Reply:
There is I wrote the test, that dialog is invisible.

Reply:
I see call of the closing only for base64. Please, test it with hands just in case :)

Reply:
I tested with hands, it work as expected